src/main/java/pvs/app/service/GithubCommitService.java

```diff
@@ -46,4 +46,16 @@ public GithubCommitDTO getLastCommit(String repoOwner, String repoName) {
         dto.setCommittedDate(githubCommit.getCommittedDate());
         return dto;
     }
+
+    public List<GithubCommitDTO> getCommitsOfSpecificBranch(String repoOwner, String repoName, String branchName) {
```

Comment:
ditto.

```suggestion
    public List<GithubCommitDTO> getGithubCommitDTOs(String repoOwner, String repoName, String branchName) {
```

But for me, name to `getCommitsOfSpecificBranch` can still know the purpose of this function, so I think this part is up to you.